src/net.h

```diff
@@ -1048,17 +1046,7 @@ class CNode
      * in CConnman::AttemptToEvictConnection. */
     std::atomic<int64_t> nLastTXTime{0};
 
-    // Ping time measurement:
-    // The pong reply we're expecting, or 0 if no pong expected.
-    std::atomic<uint64_t> nPingNonceSent{0};
-    /** When the last ping was sent, or 0 if no ping was ever sent */
-    std::atomic<std::chrono::microseconds> m_ping_start{0us};
-    // Last measured round-trip time.
-    std::atomic<int64_t> nPingUsecTime{0};
-    // Best measured round-trip time.
-    std::atomic<int64_t> nMinPingUsecTime{std::numeric_limits<int64_t>::max()};
-    // Whether a ping is requested.
-    std::atomic<bool> fPingQueued{false};
+    std::atomic<int64_t> m_best_ping_time{std::numeric_limits<int64_t>::max()};
```

Comment:
```suggestion
    std::atomic<int64_t> m_min_ping_time{std::numeric_limits<int64_t>::max()};
```
I think renaming from "min" to "best" needlessly sows confusion:

- "min" is more precise than "best" (best how? in which way?)
- getpeerinfo and its help, the GUI peers info, and -netinfo and its new help (#20764) all refer to min ping, Min Ping, mping, and minimum observed ping time

Reply:
I agree. I've changed it to `m_min_ping_time` (it's still code churn since it was previously named `nMinPingUsecTime`, but I agree with you about the consistent naming).